CMakeLists.txt

```diff
@@ -157,10 +157,7 @@ if(BUILD_VERSION STREQUAL "0.0.0")
             )
             if(NOT CALC_RELEASE_VERSION_RESULT STREQUAL 0)
                 # If python failed above, stderr would tell the user about it
-                message(WARNING
-                    "BUILD_VERSION not specified and could not be calculated\
- (script invocation failed); setting library version to 0.0.0"
-                )
+                message(FATAL_ERROR "BUILD_VERSION not specified and could not be calculated")
```

Comment:
@rcsanchez97 we were planning to make this an error [per this comment](https://jira.mongodb.org/browse/CDRIVER-3022?focusedCommentId=2187589&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2187589). This looks like the right change to me, but just wanted to run this by you.

Reply:
@kevinAlbs I concur.  However, the comment you linked to is against the C driver.  I am actually a but surprised that we did not write a corresponding C++ driver ticket.  Either way this looks good to me.